### PR TITLE
Finish Cinema Lab

### PR DESCRIPTION
**Pull request by XOctOpus1**

### CODE REVIEW

File: app/cinema/bar.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from app.people import customer
```
Comment (ArsenPidhoretskyi) [inline app/cinema/bar.py:1]: You can do:
from app.people.customer import Customer

File: app/main.py [hunk 2]
```diff
@@ -1,6 +1,22 @@
...
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(customers: list,
+                 hall_number: int,
+                 cleaner: str,
+                 movie: str) -> None:
+
+    cust = []
```
Comment (ArsenPidhoretskyi) [inline app/main.py:12]: cust is not good name rename to customers_objs

File: app/main.py [hunk 3]
```diff
@@ -1,6 +1,22 @@
...
+    clean = Cleaner(cleaner)
+
+    for person in customers:
+        cust.append(Customer(person["name"], person["food"]))
+
+    for person in cust:
+        CinemaBar.sell_product(person.food, person)
+
+    hl = CinemaHall(hall_number)
```
Comment (ArsenPidhoretskyi) [inline app/main.py:21]: hl is not clear name. Rename to cinema_hall

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 4]
```diff
@@ -1,6 +1,22 @@
...
+                 cleaner: str,
+                 movie: str) -> None:
+
+    customers_objs = []
+    clean = Cleaner(cleaner)
+
+    for person in customers:
+        customers_objs.append(Customer(person["name"], person["food"]))
+
+    for person in customers_objs:
```
Comment (vasylhnatiuk) [inline app/main.py:18]: after delete this loop

File: app/people/customer.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+class Customer:
+    def __init__(self,
+                 name: str,
+                 food: str) -> None:
+        self.name = name
+        self.food = food
+
+    def watch_movie(self,
```
Comment (vasylhnatiuk) [inline app/people/customer.py:8]:
```suggestion
    def watch_movie(self, movie: str) -> None:
```

File: app/people/cinema_staff.py [hunk 6]
```diff
@@ -0,0 +1,8 @@
+class Cleaner:
+    def __init__(self,
+                 name: str) -> None:
+        self.name = name
+
+    def clean_hall(self,
```
Comment (vasylhnatiuk) [inline app/people/cinema_staff.py:6]:
```suggestion
    def clean_hall(self, movie: str) -> None:
```

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

**Review by vasylhnatiuk — APPROVED** [review]
Good job

**Pull request closed without merging**